src/definition/AppStatus.ts

```diff
@@ -51,6 +51,13 @@ export class AppStatusUtilsDef {
                 return false;
         }
     }
+
+    public isError(status: AppStatus): boolean {
+        return [
```

Comment:
This is a micro-optimization, but wouldn't it be better to have this array created static and as a constant instead of having to be created every time this method is called?

Reply:
Indeed it would be slightly more efficient, indeed it is a micro-optimization. IMO it's clearer to read it that way, everything is inside the method's body.

If time comes we need this list elsewhere, no doubt we should extract to a static array. Now I'm more inclined to go with the "clearer" version

Reply:
Interesting, I’m of the opposite opinion. I think it is more clear to read the code when it is reading a named variable/constant that expresses what it is for than to declare it in the body of the method being called.

Reply:
Taking what you said as a rule is generally safer, but in this specific case, we have a bunch of things helping us: short method body; short method name; clear enum items named inside the array; few items inside it.

But should the situation be different, it would be worth it to go down a different path 😃